### PR TITLE
Chopping AOD files according to defined size (default = 8 GB)

### PR DESCRIPTION
**Pull request by chiarazampolli**
@catalinristea : if you could test, it would be great!

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Hi @catalinristea , 
Here what we discussed. Can you take a look and try this out?
Thanks in advance,
Chiara

File: DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh [hunk 1]
```diff
@@ -311,8 +318,48 @@ fi
...
+  echo current size = $CURRENT_SIZE
+  PERCENT=`echo "scale=2; $CURRENT_SIZE/($AOD_FILE_SIZE*10^6)*100" | bc -l`
+  echo percent = $PERCENT
+  if (( $(echo "$PERCENT < 80" | bc -l) )); then
+    AOD_LAST_BUT_ONE=`find . -name AO2D.root | sort | tail -2 | head -1`
+    echo "Too small, merging $AOD_LAST with previous file $AOD_LAST_BUT_ONE"
+    ls $PWD/$AOD_LAST > list.list
+    ls $PWD/$AOD_LAST_BUT_ONE >> list.list
+    echo "List of files for merging:"
+    more list.list
```
Comment (ktf) [inline DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh:337]:
```suggestion
    cat list.list
```

Comment (chiarazampolli) [reply DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh]: Thanks!

**Comment by chiarazampolli** [comment]
Hello @catalinristea ,
I added the fixes (the "fi" leftover from rebasing, and change 80 --> 20). 
After successful tests, we can merge.
Chiara

**Pull request merged**